libs/math/tests/unit/vectors/avx_tests.cpp

```diff
@@ -112,10 +112,9 @@ using MyTypes = ::testing::Types<fetch::vectorise::VectorRegister<float, 32>,
                                  fetch::vectorise::VectorRegister<fetch::fixed_point::fp64_t, 64>,
                                  fetch::vectorise::VectorRegister<double, 64>>;
 
-using MyFPTypes = ::testing::Types<fetch::vectorise::VectorRegister<float, 32>,
-                                   fetch::vectorise::VectorRegister<fetch::fixed_point::fp32_t, 32>,
-                                   fetch::vectorise::VectorRegister<fetch::fixed_point::fp64_t, 64>,
-                                   fetch::vectorise::VectorRegister<double, 64>>;
+using MyFPTypes =
```

Comment:
again please move definitions of groups of types for testing into math/test/unit/test_types.hpp
this might be called PrimitiveFloatTypes or something like that

Reply:
ok, however, their definition depends on whether AVX2 is enabled, I'll try to find a better solution for this

Reply:
I will also keep those as they are right now, these affect only VectorRegister types, for only the vector_tests -and used nowhere else. Furthermore, they are under an platform specific #ifdef AVX2, and would need modification for other architectures. I will investigate another way to fix this.